### PR TITLE
if there are no remote syslog servers, just do local config

### PR DESCRIPTION
**Pull request by rdoorn**
do not break if your not using remote syslog hosts, just don't write a config file for remote servers.
so you can still use the cookbook to manage your local config or if you have not yet created any syslog servers in your environment yet.

### CODE REVIEW

File: recipes/client.rb [hunk 1]
```diff
@@ -65,20 +65,18 @@ def chef_solo_search_installed?
   rsyslog_servers += node['rsyslog']['custom_remote']
 end
 
-if rsyslog_servers.empty?
-  Chef::Application.fatal!('The rsyslog::client recipe was unable to determine the remote syslog server. Checked both the server_ip attribute and search!')
```
Comment (tas50) [inline recipes/client.rb]: We should probably turn this into an error vs. fatal, but it's a good thing to leave it present.

File: metadata.rb [hunk 2]
```diff
@@ -4,7 +4,7 @@
 license           'Apache 2.0'
 description       'Installs and configures rsyslog'
 long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version           '2.1.1'
+version           '2.1.2'
```
Comment (tas50) [inline metadata.rb]: The version should not be rev'd here.  I'll handle the version bumps during the release phase.

**Comment by rdoorn** [comment]
Thanks @tas50 I've updated the cookbook according to the comments, all valid points.

**Comment by tas50** [comment]
Thanks for fixing this up.  rsyslog::client is more useful out of the box now.

**Pull request merged**